### ISSUE TITLE
Missing tracepoints and code sections will cause runtime panic

### ISSUE THREAD

**Issue by ZekeMedley** [cloudflare/ebpf_exporter#103]
A slightly modified version of the timers example:

```yaml
programs:
  - name: timers
    metrics:
      counters:
        - name: timer_start_total
          help: Timers fired in the kernel
          table: counts
          labels:
            - name: function
              size: 8
              decoders:
                - name: ksym
    # tracepoints:
    #   timer:timer_start: tracepoint__timer__timer_start
    # code: |
    #   BPF_HASH(counts, u64);

    #   // Generates function tracepoint__timer__timer_start
    #   TRACEPOINT_PROBE(timer, timer_start) {
    #       counts.increment((u64) args->function);
    #       return 0;
    #   }
```

Will cause a `panic: runtime error: index out of range [0] with length 0` when run.

<details>
<summary>Full output</summary>
<br>
<pre>
panic: runtime error: index out of range [0] with length 0

goroutine 29 [running]:
github.com/iovisor/gobpf/bcc.(*TableIterator).Next(0xc0001c0eb0, 0xc0001c0eb0)
	/cfsetup_build/prometheus-ebpf-exporter/tmp/build/cloudflare-ebpf_exporter-cf86dcc/vendor/github.com/iovisor/gobpf/bcc/table.go:283 +0x3e5
github.com/cloudflare/ebpf_exporter/exporter.(*Exporter).tableValues(0xc0001c0a50, 0xc0001dcc00, 0xc0001baf18, 0x6, 0xc000230bd0, 0x1, 0x1, 0x7f3121bcd190, 0xaf, 0x7f3123544900, ...)
	/cfsetup_build/prometheus-ebpf-exporter/tmp/build/cloudflare-ebpf_exporter-cf86dcc/exporter/exporter.go:242 +0xfa
github.com/cloudflare/ebpf_exporter/exporter.(*Exporter).collectCounters(0xc0001c0a50, 0xc0001bf380)
	/cfsetup_build/prometheus-ebpf-exporter/tmp/build/cloudflare-ebpf_exporter-cf86dcc/exporter/exporter.go:151 +0x1b2
github.com/cloudflare/ebpf_exporter/exporter.(*Exporter).Collect(0xc0001c0a50, 0xc0001bf380)
	/cfsetup_build/prometheus-ebpf-exporter/tmp/build/cloudflare-ebpf_exporter-cf86dcc/exporter/exporter.go:143 +0x398
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
	/cfsetup_build/prometheus-ebpf-exporter/tmp/build/cloudflare-ebpf_exporter-cf86dcc/vendor/github.com/prometheus/client_golang/prometheus/registry.go:443 +0x12b
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
	/cfsetup_build/prometheus-ebpf-exporter/tmp/build/cloudflare-ebpf_exporter-cf86dcc/vendor/github.com/prometheus/client_golang/prometheus/registry.go:454 +0x5ce
</pre>
</details>

This is clearly bad input so it's not a huge deal but it might be nice to surface a warning if the `code` section is missing.

**Comment by ZekeMedley** [comment]
For a more realistic example of this happening: this came up because I messed up some indentation which caused the code and tracepoints sections of the config to not get generated correctly.

**Comment by bobrik** [comment]
Fixed in #104, thanks!
